### DISCUSSION TITLE
Render only first level (parent) of collection

### DISCUSSION THREAD

**Discussion by MizuBishi** [statamic/cms#4494] [Q&A] [answered]
Hi Guys
I am new to statamic and I need help. I would like to render only the first level (parent) of my collection. 
Example:
- a
  * 1
  * 2
  * 3
- b
  * 4
  * 5
- c
  * 6

I would like to get `a, b, c` but with 
```
{{ collection:teams }}
    {{ title }}
{{ /collection:teams }}
```
I get all Levels. Thank u so much for your help!!!

**Comment by jasonvarga** [comment, marked as answer]
Use the `nav` tag with a `max_depth` parameter.

```
{{ nav:collection:teams max_depth="1" }}
  {{ title }}
{{ /nav:collection:teams }}
```

**Reply by MizuBishi** [reply]
Hi @jasonvarga works like a charm! Thank u so much!
